biogeochem/EDPhysiologyMod.F90

```diff
@@ -2314,18 +2326,22 @@ subroutine CWDOut( litt, fragmentation_scaler, nlev_eff_decomp )
     integer :: c
     integer :: ilyr
     integer :: dcmpy
-    integer :: numlevsoil
     !----------------------------------------------------------------------
 
+
+    ! Above ground litters are associated with the top soil layer temperature and
+    ! moisture scalars and as such the top-layer fragmentation scalar index of 1
+    ! is used for ag_cwd_frag and root_fines_frag calculations.
+
     do c = 1,ncwd  
 
        litt%ag_cwd_frag(c)   = litt%ag_cwd(c) * SF_val_max_decomp(c) * &
-             years_per_day * fragmentation_scaler
+             years_per_day * fragmentation_scaler(1)
```

Comment:
@glemieux to make this less error prone for changes suggest changing this, so if a user makes changes it will be in one spot. To that end, please confirm I did not miss an index use. (I found two places)
```suggestion
             years_per_day * fragmentation_scaler(soil_layer_index)
```
```suggestion
             years_per_day * fragmentation_scaler(1)
```

Reply:
@jkshuman it looks like the second suggestion did not use the defined variable soil_layer_index

Reply:
thanks @rgknox I edited my post. that was odd, it was the same suggestion in there twice. looked over other suggestions. 
@glemieux please sanity check.